build.zig

```diff
@@ -94,12 +95,12 @@ pub fn build(b: *Builder) !void {
     }
 
     const debug_step = b.step("debug", "Debug with gdb");
+    const symbol_file_arg = try std.mem.join(b.allocator, " ", [_][]const u8{ "symbol-file", elf_path });
     const debug_cmd = b.addSystemCommand([_][]const u8{
         "gdb",
         "-ex",
-        "symbol-file",
     });
```

Comment:
why not add the arguments here. So have:
const debug_cmd = b.addSystemCommand([_][]const u8{
    "gdb",
    "-ex",
    "symbol-file",
    elf_path ,
});

Reply:
"symbol-file" and the path have to be in the same string, so I could do
```zig
const debug_cmd = b.addSystemCommand([_][]const u8{
    "gdb",
    "-ex",
    symbol_file_arg,
});
```
instead

Reply:
which ever look neater/clearer